loadtest.py

```diff
@@ -110,4 +110,5 @@ def test_crash_greater_than_20mb_uncompressed():
     resp = utils.post_crash(URL_SERVER, crash_payload)
     print("test_crash_greater_than_20mb_uncompressed: HTTP %s" % resp.status_code)
     # Verify HTTP 413
-    assert 413 == resp.status_code
+
+    assert 413 == resp.status_code, "Failure - HTTP %s, Page text: %s" % (resp.status_code, resp.text)
```

Comment:
That's covered in the system test already. We're just writing load tests here. Everything else should be covered in the unit tests and system tests:

https://github.com/mozilla/antenna/tree/master/tests